app/code/Magento/CatalogImportExport/Model/Import/Product/CategoryProcessor.php

```diff
@@ -204,4 +221,15 @@ public function getCategoryById($categoryId)
     {
         return isset($this->categoriesCache[$categoryId]) ? $this->categoriesCache[$categoryId] : null;
     }
+
+    /**
+     * Standardize a string by lowercase-ing it
+     *
+     * @param string $string
+     * @return string
+     */
+    protected function standardizeString($string)
```

Comment:
Do you still need a method which just calls `strtolower`? I guess it is redundant now